apps/src/templates/sectionProgress/multiGridConstants.js

```diff
@@ -55,11 +55,18 @@ export const progressStyles = {
     backgroundColor: color.table_header,
   },
   icon: {
-    paddingLeft: '10px',
-    paddingRight: '10px',
-    paddingTop: '12px',
+    paddingLeft: 10,
+    paddingRight: 10,
+    paddingTop: 12,
+    width: 38,
+    fontSize: 20,
+  },
+  unpluggedIcon: {
     width: 38,
```

Comment:
Might be more readable to leave padding as is, and then for each the width can be `PILL_BUBBLE_WIDTH`, `PROGRESS_BUBBLE_WIDTH`, or `DIAMOND_BUBBLE_WIDTH` as defined above?

Reply:
<img width="810" alt="screen shot 2018-04-25 at 7 56 00 am" src="https://user-images.githubusercontent.com/12300669/39254173-53a62c44-485e-11e8-983e-e05ab508013a.png">
If you go that route, you end up with ^ because `PROGRESS_BUBBLE_WIDTH` doesn't account for the L/R padding that each icon needs.  I have to either adjust the padding anyway to play nicely with the new width or use the width, but do some math to actually make it the width we want.